indico/modules/events/tracks/operations.py

```diff
@@ -43,3 +44,25 @@ def update_program(event, data):
     track_settings.set_multi(event, data)
     logger.info('Program of %r updated by %r', event, session.user)
     event.log(EventLogRealm.management, EventLogKind.change, 'Tracks', 'The program has been updated', session.user)
+
+
+def create_track_group(event, data):
+    track_group = TrackGroup()
```

Comment:
Shouldn't the `track_group` be associated with the `event` here?

Reply:
Yup, forgot to add it after I created a relationship between `TrackGroup` and `Event`.

Reply:
Updated

Reply:
Okay, I'm adding it in my PR. Just letting you know.

Reply:
I have just updated my PR ;) you can update